website/content/about/working-groups/governance.md

```diff
@@ -4,17 +4,114 @@ linkTitle: Governance
 url: /about/governance/
 ---
 
+{{% blocks/lead color="primary" align="left" %}}
 
-We aim to assist all CNCF projects with implementing good open source governance
-practices.
+Want to implement good open source governance? We can assist you with best practices, resources, and guidance on how!
 
-We meet once a month.
-Go to <a href="https://www.cncf.io/calendar/">cncf.io/calendar</a> and enter "Governance WG" in the search box to see when we meet next.
-    
-* [Zoom](https://zoom.us/my/cncftagcontributorstrategy?pwd=TnI0WU9Eb2I1RlRWdkl1R0k1WkZXUT09) passcode: `77777`
-* [Meeting Minutes and Agenda](https://docs.google.com/document/d/1P9tQgCM6OwDHd1F8UnWuauL4KDVTMTp49_n64_w8nrs/edit)
+Read our [Charter](https://github.com/cncf/tag-contributor-strategy/blob/main/governance/README.md)
 
-Discussion happens on the [mailing list] or on #tag-contributor-strategy on [Slack].
+{{% /blocks/lead %}}
 
-[mailing list]: https://lists.cncf.io/g/cncf-tag-contributor-strategy
-[Slack]: https://slack.cncf.io/
+<div class="section-group">
+
+{{% blocks/section color="dark" %}}
+
+The Governance Working Group assists CNCF projects with implementing good open source governance practices.
+
+We provide best practices and resources for managing and interacting with contributors to CNCF projects.
+{{% /blocks/section %}}
+
+</div>
+
+<div class="section-group">
+
+{{% blocks/section color="secondary" %}}
+
+## Goals
+
+<div class="text-left">
+
+* Create, share, and maintain documentation on implementing good open source governance for CNCF projects.
+* Coach and advise CNCF projects, and prospective projects, leaders in implementing and improving governance.
+* Advise the CNCF TOC on updating governance criteria for acceptance and graduation.
+
+</div>
+
+{{% /blocks/section %}}
+
+</div>
+
+<div class="section-group">
+{{< blocks/section color="primary" >}}
+
+## How your community can benefit
+
+<div class="text-left">
+
+* Have a look at our [Governance Guide](/maintainers/governance/) to see best practices, get advice and have inspiration while setting up or improving your project and community governance
+* Copy and modify one of our codified [Governance Templates](/maintainers/templates/governance-intro/) depending on your community and project
+* Get assistance on governance related issues
+
+</div>
+
+{{% /blocks/section %}}
+</div>
+
+<div class="section-group">
+{{< blocks/section color="primary" >}}
+
+## How you can contribute
+
+<div class="text-left">
```

Comment:
I can't figure out why the text for this section is showing up on the same line as the heading. I'm going to merge and we can fix that visual quirk later though.